tests/README.rst

```diff
@@ -0,0 +1,51 @@
+Unit tests
+==========
+
+To run unit tests for the sphinx-immaterial theme, there must first be dependencies installed.
+Install the dependencies by executing the following from the repo root folder:
+
+.. code-block:: shell
+
+    pip install -r tests/requirements.txt
+
+The unit tests also require `graphviz <https://graphviz.org/download/>`_ installed.
+
+The sphinx-immaterial theme should also be installed. For developer purposes, we recommend
+using an editable install, but this also requires that theme assets be assembled (as is done for
+distribution).
+
+.. code-block:: shell
+
+    npm install
+    npm run build
+    pip install -e .
```

Comment:
I believe `pip install -e` also does the npm build automatically.  Is that not correct?

Reply:
No, we had someone ask recently why sphinx failed to find the html templates when it was using an editable install.

Reply:
Looks like the issue is that `pip install -e .` no longer runs the `develop` command, which is what we were relying on in setup.py to do the nodejs build.  It looks like there is a new mechanism that can be used to handle editable builds, but it is not clear to me exactly how to use it:

https://setuptools.pypa.io/en/latest/userguide/extension.html#supporting-sdists-and-editable-installs-in-build-sub-commands

Reply:
I honestly don't mind the way it works now. It cuts down on the initial editable install time significantly.

Although, I do understand the convenience for newcomers not having to remember the npm build step. Should this become a separate issue?

Reply:
I created an issue.  You don't need to re-run `pip install -e .` that often -- that is, after all, the whole point of an editable install.  So I think it is definitely desirable for it to just work by default.